From ab6ecf385f68633421c43e46ccde83b589bf8352 Mon Sep 17 00:00:00 2001
From: Stefan Lange <timo.herbrecher@tq-group.com>
Date: Tue, 29 Mar 2016 14:55:43 +0200
Subject: [PATCH] set RGMII delay values in ethernet phys located on the
 STKT104x starterkit

---
 board/tqc/tqmt1042/eth.c | 51 ++++++++++++++++++++++++++++++++++++++++++++++++
 1 file changed, 51 insertions(+)

diff --git a/board/tqc/tqmt1042/eth.c b/board/tqc/tqmt1042/eth.c
index e6ded86..a21492f 100644
--- a/board/tqc/tqmt1042/eth.c
+++ b/board/tqc/tqmt1042/eth.c
@@ -12,6 +12,57 @@
 #include <fsl_mdio.h>
 #include <malloc.h>
 #include <asm/fsl_dtsec.h>
+#include <phy.h>
+
+/*
+ * Set RGMII delay values in ethernet phys
+ * located on the STKT104x starterkit
+ */
+int board_phy_config(struct phy_device *phydev)
+{
+
+#define STKT104X_EC1_PHYAD            0x0E
+#define STKT104X_EC2_PHYAD            0x05
+#define MII_DP83867_REGCR             0x000D
+#define MII_DP83867_ADDAR             0x000E
+#define MII_DP83867_REGCR_AD          0x001F
+#define MII_DP83867_REGCR_DAT         0x401F
+#define MII_DP83867_RGMIICTL          0x0032
+#define MII_DP83867_RGMIIDCTL         0x0086
+
+	if (phydev->drv->config)
+		phydev->drv->config(phydev);
+
+	if (phydev->addr == STKT104X_EC1_PHYAD) {
+		/* set EC1 RGMII delay (extended write) */
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_AD);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,MII_DP83867_RGMIIDCTL);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_DAT);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,0x0097);
+
+		/* activate EC1 RGMII TX and RX delay (extended write) */
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_AD);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,MII_DP83867_RGMIICTL);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_DAT);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,0x00D3);
+	}
+
+	if (phydev->addr == STKT104X_EC2_PHYAD) {
+		/* set EC2 RGMII delay (extended write) */
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_AD);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,MII_DP83867_RGMIIDCTL);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_DAT);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,0x0097);
+
+		/* activate EC2 RGMII TX and RX delay (extended write) */
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_AD);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,MII_DP83867_RGMIICTL);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_REGCR,MII_DP83867_REGCR_DAT);
+		phy_write(phydev,MDIO_DEVAD_NONE,MII_DP83867_ADDAR,0x00D3);
+	}
+
+	return 0;
+}
 
 int board_eth_init(bd_t *bis)
 {
-- 
1.9.1

