From 388479d1d9151a66e1dd93c8d2be4631c1f2b901 Mon Sep 17 00:00:00 2001
From: Timo Herbrecher <t.herbrecher@gateware.de>
Date: Wed, 25 May 2016 14:32:51 +0200
Subject: [PATCH] Added RCW mkimage defines for NOR flash boot

---
 include/configs/TQMT1042.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/TQMT1042.h b/include/configs/TQMT1042.h
index f056e4d..3b0034d 100644
--- a/include/configs/TQMT1042.h
+++ b/include/configs/TQMT1042.h
@@ -61,6 +61,9 @@
 #endif
 #define CONFIG_SPL_MMC_BOOT
 #endif
+#else
+#define CONFIG_FSL_RCW_BIN
+#define CONFIG_SYS_FSL_RCW $(SRCTREE)/board/tqc/tqmt1042/tqmt1042_rcw.cfg
 #endif
 
 /* High Level Configuration Options */
-- 
1.9.1

