From 138c843de24e61ea40fc700464933718f56827e0 Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Thu, 16 Jun 2016 08:54:05 +0200
Subject: [PATCH] set TQMT1042 default environment

---
 include/configs/TQMT1042.h | 113 +++++++++++++++++++++++++--------------------
 1 file changed, 62 insertions(+), 51 deletions(-)

diff --git a/include/configs/TQMT1042.h b/include/configs/TQMT1042.h
index 79c0b92..65b495a 100644
--- a/include/configs/TQMT1042.h
+++ b/include/configs/TQMT1042.h
@@ -631,82 +631,93 @@
 /*
  * Environment Configuration
  */
-#define CONFIG_ROOTPATH		"/opt/nfsroot"
-#define CONFIG_BOOTFILE		"uImage"
-#define CONFIG_UBOOTPATH	"u-boot.bin"	/* U-Boot image on TFTP server*/
+#define CONFIG_ROOTPATH		"/srv/nfs/rootfs-64b"
+#define CONFIG_BOOTFILE		"boot/uImage_tqmt1042"					/* uImage on NFS server*/
+#define CONFIG_UBOOTPATH	"u-boot/u-boot_tqmt1042.bin"	/* U-Boot image on TFTP server*/
 
 /* default location for tftp and bootm */
-#define CONFIG_LOADADDR		1000000
+#define CONFIG_LOADADDR		0x4000000
 
-#define CONFIG_BOOTDELAY	10	/*-1 disables auto-boot*/
+#define CONFIG_BOOTDELAY	5	/*-1 disables auto-boot*/
 
 #define CONFIG_BAUDRATE	115200
 
 #define __USB_PHY_TYPE	utmi
-#define RAMDISKFILE	"tqmt1042/ramdisk.uboot"
-#define FDTFILE		"tqmt1042/tqmt1042.dtb"
+#define RAMDISKFILE	"uboot/ramdisk.ext2.gz.u-boot"
+#define FDTFILE		"boot/tqmt1042.dtb"
 
 #ifdef CONFIG_FSL_DIU_FB
-#define DIU_ENVIRONMENT "video-mode=fslfb:1024x768-32@60,monitor=dvi" /* TODO */
+#define DIU_ENVIRONMENT "video-mode=fslfb:1024x768-24@60"
 #else
 #define DIU_ENVIRONMENT
 #endif
 
-/* TODO */
 #define	CONFIG_EXTRA_ENV_SETTINGS				\
-	"hwconfig=fsl_ddr:bank_intlv=cs0_cs1;"			\
-	"usb1:dr_mode=host,phy_type=" __stringify(__USB_PHY_TYPE) ";"\
-	"usb2:dr_mode=host,phy_type=" __stringify(__USB_PHY_TYPE) "\0"\
-	"netdev=eth0\0"						\
+	"hwconfig=fsl_ddr:bank_intlv=cs0_cs1;"              	\
+    "usb1:dr_mode=host,phy_type=" __stringify(__USB_PHY_TYPE) ";"\
+    "usb2:dr_mode=host,phy_type=" __stringify(__USB_PHY_TYPE) "\0"\
+	"netdev=fm1-mac3\0"					\
 	"video-mode=" __stringify(DIU_ENVIRONMENT) "\0"		\
 	"uboot=" __stringify(CONFIG_UBOOTPATH) "\0"		\
 	"ubootaddr=" __stringify(CONFIG_SYS_TEXT_BASE) "\0"	\
-	"tftpflash=tftpboot $loadaddr $uboot && "		\
-	"protect off $ubootaddr +$filesize && "			\
-	"erase $ubootaddr +$filesize && "			\
-	"cp.b $loadaddr $ubootaddr $filesize && "		\
-	"protect on $ubootaddr +$filesize && "			\
-	"cmp.b $loadaddr $ubootaddr $filesize\0"		\
+	"ubootaddr_flsh=0xeff40000\0"				\
+	"uboot_size=0xc0000\0"					\
 	"consoledev=ttyS0\0"					\
-	"ramdiskaddr=2000000\0"					\
+	"ramdiskaddr=0x2000000\0"				\
+	"ramdiskaddr_flsh=0xe8840000\0"				\
 	"ramdiskfile=" __stringify(RAMDISKFILE) "\0"		\
-	"fdtaddr=c00000\0"					\
+	"ramdisk_size=0x2500000\0"				\
+	"fdtaddr=0xe00000\0"					\
+	"fdtaddr_flsh=0xe8820000\0"				\
 	"fdtfile=" __stringify(FDTFILE) "\0"			\
-	"bdev=sda3\0"
-
-#define CONFIG_LINUX                       \
-	"setenv bootargs root=/dev/ram rw "            \
-	"console=$consoledev,$baudrate $othbootargs;"  \
-	"setenv ramdiskaddr 0x02000000;"               \
-	"setenv fdtaddr 0x00c00000;"		       \
-	"setenv loadaddr 0x1000000;"		       \
-	"bootm $loadaddr $ramdiskaddr $fdtaddr"
-
-#define CONFIG_HDBOOT					\
-	"setenv bootargs root=/dev/$bdev rw "		\
-	"console=$consoledev,$baudrate $othbootargs;"	\
-	"tftp $loadaddr $bootfile;"			\
-	"tftp $fdtaddr $fdtfile;"			\
+	"baudrate=115200\0"					\
+	"ethact=FM1@DTSEC4\0"					\
+	"ethprime=FM1@DTSEC4\0"					\
+	"kerneladdr_flsh=e8020000\0"				\
+	"fman_ucodeaddr_flsh=eff00000\0"			\
+	"rcwfile=rcw/fsl_rcw_tqmt1042.bin\0"			\
+	"rcw_addr_flsh=0xe8000000\0"				\
+	"upd_uboot=if tftp $uboot; then echo updating "		\
+	"u-boot...; "						\
+	"protect off $ubootaddr_flsh +$uboot_size; "		\
+	"erase $ubootaddr_flsh +$uboot_size; " 			\
+	"cp.b $loadaddr $ubootaddr_flsh $uboot_size; " 		\
+	"else echo u-boot file not found!; fi\0"		\
+	"upd_rcw=if tftp $loadaddr $rcwfile; then echo "	\
+	"updating rcw...; erase 0xe8000000 +1; "		\
+	"cp.b $loadaddr $rcw_addr_flsh $filesize; " 		\
+	"else echo RCW file not found!; fi\0"			\
+	"flashboot=" CONFIG_HDBOOT "\0"				\
+	"mmcboot=" CONFIG_MMCBOOT "\0"
+
+#define CONFIG_LINUX
+
+#define CONFIG_HDBOOT                       			\
+	"setenv bootargs root=/dev/ram rw "            		\
+	"console=$consoledev,$baudrate $othbootargs "  		\
+	"ramdisk_size=$ramdisk_size; "	    			\
+	"bootm $kerneladdr_flsh $ramdiskaddr_flsh "		\
+	"$fdtaddr_flsh"
+
+#define CONFIG_MMCBOOT						\
+	"setenv bootargs root=/dev/mmcblk0p1 rw "		\
+	"rootfstype=ext2 rootwait "				\
+	"ip=$ipaddr:$serverip:$gatewayip:$netmask:$hostname:$netdev:off " \
+	"console=$consoledev,$baudrate $othbootargs; "		\
+	"ext2load mmc 0:1 $fdtaddr $fdtfile; "			\
+	"ext2load mmc 0:1 $loadaddr $bootfile; " 		\
 	"bootm $loadaddr - $fdtaddr"
 
-#define CONFIG_NFSBOOTCOMMAND			\
-	"setenv bootargs root=/dev/nfs rw "	\
-	"nfsroot=$serverip:$rootpath "		\
+#define CONFIG_NFSBOOTCOMMAND					\
+	"setenv bootargs root=/dev/nfs rw "			\
+	"nfsroot=$serverip:$rootpath "				\
 	"ip=$ipaddr:$serverip:$gatewayip:$netmask:$hostname:$netdev:off " \
-	"console=$consoledev,$baudrate $othbootargs;"	\
-	"tftp $loadaddr $bootfile;"		\
-	"tftp $fdtaddr $fdtfile;"		\
+	"console=$consoledev,$baudrate $othbootargs; "		\
+	"nfs $loadaddr $serverip:$rootpath/$bootfile; "		\
+	"nfs $fdtaddr $serverip:$rootpath/$fdtfile; "		\
 	"bootm $loadaddr - $fdtaddr"
 
-#define CONFIG_RAMBOOTCOMMAND				\
-	"setenv bootargs root=/dev/ram rw "		\
-	"console=$consoledev,$baudrate $othbootargs;"	\
-	"tftp $ramdiskaddr $ramdiskfile;"		\
-	"tftp $loadaddr $bootfile;"			\
-	"tftp $fdtaddr $fdtfile;"			\
-	"bootm $loadaddr $ramdiskaddr $fdtaddr"
-
-#define CONFIG_BOOTCOMMAND		CONFIG_LINUX
+#define CONFIG_BOOTCOMMAND		CONFIG_HDBOOT
 
 #ifdef CONFIG_SECURE_BOOT
 #include <asm/fsl_secure_boot.h>
-- 
1.9.1

