From e11b5705aced0ff449b7d681d77d82fb5c4d52a0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Konrad=20Gr=C3=A4fe?= <k.graefe@gateware.de>
Date: Mon, 10 Oct 2016 10:45:53 +0200
Subject: [PATCH] TQMT1042: Add SD card boot command

---
 include/configs/TQMT1042.h | 13 +++++++++++++
 1 file changed, 13 insertions(+)

diff --git a/include/configs/TQMT1042.h b/include/configs/TQMT1042.h
index 313c189..adb93c6 100644
--- a/include/configs/TQMT1042.h
+++ b/include/configs/TQMT1042.h
@@ -717,7 +717,20 @@
 	"nfs $fdtaddr $serverip:$rootpath/$fdtfile; "		\
 	"bootm $loadaddr - $fdtaddr"
 
+#define CONFIG_SDCARDBOOTCOMMAND \
+	"setenv bootargs root=/dev/mmcblk0p2 rw rootwait " \
+	"ip=$ipaddr:$serverip:$gatewayip:$netmask:$hostname:$netdev:off " \
+	"console=$consoledev,$baudrate $othbootargs; "		\
+	"fatload mmc 0:1 $loadaddr uImage; " \
+	"fatload mmc 0:1 $fdtaddr tqmt1042.dtb; " \
+	"bootm $loadaddr - $fdtaddr"
+
+
+#ifdef CONFIG_SDCARD
+#define CONFIG_BOOTCOMMAND		CONFIG_SDCARDBOOTCOMMAND
+#else
 #define CONFIG_BOOTCOMMAND		CONFIG_HDBOOT
+#endif
 
 #ifdef CONFIG_SECURE_BOOT
 #include <asm/fsl_secure_boot.h>
-- 
1.9.1

