From fda7c2d452175b39e498a7f56ace7bdc99de3d49 Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Wed, 12 Oct 2016 13:15:30 +0200
Subject: [PATCH] [cosmetic] TQMT1042: tqmt1042.c: split comment into two lines

---
 board/tqc/tqmt1042/tqmt1042.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/board/tqc/tqmt1042/tqmt1042.c b/board/tqc/tqmt1042/tqmt1042.c
index ab92469..d55f82b 100644
--- a/board/tqc/tqmt1042/tqmt1042.c
+++ b/board/tqc/tqmt1042/tqmt1042.c
@@ -214,7 +214,8 @@ int board_early_init_r(void)
 #endif
 
 #ifdef CONFIG_SYS_TQMT1042_PLL1_CLKGEN_AUTOREPROG
-	/* read RCW SRDS_PLL_REF_CLK_SEL_S1, to detemine which PLL1 reference clock frequency is required */
+	/* read RCW SRDS_PLL_REF_CLK_SEL_S1
+	 * to detemine which PLL1 reference clock frequency is required */
 	srds_pll_ref_clk_sel_s1 = in_be32(&gur->rcwsr[5]) &
 			FSL_CORENET2_RCWSR5_SRDS_PLL_REF_CLK_SEL_S1_PLL1;
 	srds_pll_ref_clk_sel_s1 >>= FSL_CORENET2_RCWSR5_SRDS_PLL_REF_CLK_SEL_S1_PLL1_SHIFT;
-- 
1.9.1

