From ace9e71de74776f968e70edf21e69559431c70b7 Mon Sep 17 00:00:00 2001
From: Stefan Lange <timo.herbrecher@tq-group.com>
Date: Tue, 29 Mar 2016 12:44:28 +0200
Subject: [PATCH] add missing SerDes RCW configuration option 0x88 in u-boot

---
 arch/powerpc/cpu/mpc85xx/t1040_serdes.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/powerpc/cpu/mpc85xx/t1040_serdes.c b/arch/powerpc/cpu/mpc85xx/t1040_serdes.c
index d5dccd5..c3ea10e 100644
--- a/arch/powerpc/cpu/mpc85xx/t1040_serdes.c
+++ b/arch/powerpc/cpu/mpc85xx/t1040_serdes.c
@@ -33,6 +33,8 @@ static u8 serdes_cfg_tbl[][SRDS_MAX_LANES] = {
 		PCIE2, PCIE2, SGMII_FM1_DTSEC4, SGMII_FM1_DTSEC5},
 	[0x87] = {PCIE1, SGMII_FM1_DTSEC3, SGMII_FM1_DTSEC1, SGMII_FM1_DTSEC2,
 		PCIE2, PCIE3, PCIE4, SGMII_FM1_DTSEC5},
+	[0x88] = {PCIE1, SGMII_FM1_DTSEC3, SGMII_FM1_DTSEC1, SGMII_FM1_DTSEC2,
+		PCIE2, PCIE3, SATA2, SATA1},
 	[0x89] = {PCIE1, SGMII_SW1_MAC3, SGMII_SW1_MAC1, SGMII_SW1_MAC2,
 		PCIE2, PCIE3, SGMII_SW1_MAC4, SATA1},
 	[0x8D] = {PCIE1, SGMII_SW1_MAC3, SGMII_SW1_MAC1, SGMII_SW1_MAC2,
-- 
1.9.1

