From 21f7dcd922c9bdf68d517354080c3da08d440f1b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Konrad=20Gr=C3=A4fe?= <k.graefe@gateware.de>
Date: Thu, 2 Jun 2016 15:51:23 +0200
Subject: [PATCH] fsl_rcw.bin: Fix stand-alone build

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index d535486..41a3c41 100644
--- a/Makefile
+++ b/Makefile
@@ -901,7 +901,7 @@ u-boot-spl.kwb: u-boot.img spl/u-boot-spl.bin FORCE
 
 MKIMAGEFLAGS_fsl_rcw.bin = -n $(srctree)/$(CONFIG_SYS_FSL_RCW:"%"=%) -T rcwimage
 
-fsl_rcw.bin: $(srctree)/$(CONFIG_SYS_FSL_RCW:"%"=%) FORCE
+fsl_rcw.bin: tools $(srctree)/$(CONFIG_SYS_FSL_RCW:"%"=%) FORCE
 	$(call if_changed,mkimage)
 
 MKIMAGEFLAGS_u-boot-dtb.img = $(MKIMAGEFLAGS_u-boot.img)
-- 
1.9.1

