From afe639433921303fe11fead35b86bf1b2d87445f Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Thu, 7 Dec 2017 11:46:43 +0100
Subject: [PATCH] TQMT1042: remove unused DEBUG define

Signed-off-by: Stefan Lange <s.lange@gateware.de>
---
 include/configs/TQMT104x.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/configs/TQMT104x.h b/include/configs/TQMT104x.h
index 4c8c114..86bf587 100644
--- a/include/configs/TQMT104x.h
+++ b/include/configs/TQMT104x.h
@@ -10,7 +10,6 @@
 
 #include <linux/kconfig.h>
 
-#undef DEBUG
 /*
  * TQMT104x board configuration file
  */
-- 
1.9.1

