From 348d0ff398c0a53cd0ff8e33fd84a5120543b0a5 Mon Sep 17 00:00:00 2001
From: Stefan Lange <s.lange@gateware.de>
Date: Thu, 24 Nov 2016 10:21:02 +0100
Subject: [PATCH] TQMT1040: add TQMT1040 machine definition in corenet_generic

Signed-off-by: Stefan Lange <s.lange@gateware.de>
---
 arch/powerpc/platforms/85xx/corenet_generic.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/platforms/85xx/corenet_generic.c b/arch/powerpc/platforms/85xx/corenet_generic.c
index e51aaa9..50d4bf6 100644
--- a/arch/powerpc/platforms/85xx/corenet_generic.c
+++ b/arch/powerpc/platforms/85xx/corenet_generic.c
@@ -163,6 +163,7 @@ static const char * const boards[] __initconst = {
 	"keymile,kmcoge4",
 	"varisys,CYRUS",
 	"tqc,TQMT1042",
+	"tqc,TQMT1040",
 	NULL
 };
 
-- 
1.9.1

